debian/changelog

```diff
@@ -1,3 +1,65 @@
+citus (6.2.0.citus-1) stable; urgency=low
+
+  * Increases SQL subquery coverage by pushing down more kinds of queries
+
+  * Adds CustomScan API support to allow read-only transactions
+
+  * Adds support for `CREATE/DROP INDEX CONCURRENTLY`
+
+  * Adds support for `ALTER TABLE ... ADD CONSTRAINT`
+
+  * Adds support for `ALTER TABLE ... RENAME COLUMN`
+
+  * Adds support for `DISABLE/ENABLE TRIGGER ALL`
```

Comment:
This file is plaintext, not Markdown, so remove any formatting when copy-pasting from the project's `CHANGELOG.md`…